engine/src/main/java/org/hibernate/validator/constraints/kor/KorRRN.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.constraints.kor;
+
+import static java.lang.annotation.ElementType.ANNOTATION_TYPE;
+import static java.lang.annotation.ElementType.CONSTRUCTOR;
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE_USE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+import static org.hibernate.validator.constraints.kor.KorRRN.ValidateCheckDigit.NEVER;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Checks that the annotated character sequence is a valid Korean resident registration number.
+ *
+ * @author Taewoo Kim
+ * @see <a href="https://ko.wikipedia.org/wiki/%EC%A3%BC%EB%AF%BC%EB%93%B1%EB%A1%9D%EB%B2%88%ED%98%B8">Korean resident registration number</a>
+ */
+
+@Documented
+@Constraint(validatedBy = {})
+@Target({ METHOD, FIELD, ANNOTATION_TYPE, CONSTRUCTOR, PARAMETER, TYPE_USE })
+@Retention(RUNTIME)
+public @interface KorRRN {
+
+	String message() default "{org.hibernate.validator.constraints.kor.KorRRN.message}";
+
+	Class<?>[] groups() default {};
+
+	Class<? extends Payload>[] payload() default {};
+
+	ValidateCheckDigit validateCheckDigit() default NEVER;
+
+	enum ValidateCheckDigit {
+		/**
+		 * Inspect the following lines
+		 * 1. Length of RRN
+		 * 2. Validity of Gender-Digit in RRN
+		 * 3. Validity of date in RRN
+		 */
```

Comment:
Javadoc will render this to a single line, so maybe let's update it to the list as: 
```suggestion
		/**
		 * Perform the checks on:
		 * <ul>
		 *     <li>The length of an RRN</li>
		 *     <li>The validity of the Gender-Digit in an RRN</li>
		 *     <li>The validity of the date in an RRN</li>
		 * </ul>
		 */
```
and do the same for the other constant